viite-UI/src/view/Styler.js

```diff
@@ -34,48 +36,53 @@
     var opacityMultiplier = 1;
 
     var generateStrokeColor = function (roadClass, anomaly, constructionType, roadLinkType, gapTransfering, roadLinkSource, roadId) {
+      if (applicationModel.getSelectedLayer() !== 'linkProperty' && roadClass !== 99) {
+        setOpacityMultiplier(0.2);
+      } else {
+        setOpacityMultiplier(1);
+      }
       var unsavedRoadId = 0;
       if (roadLinkSource === LINKSOURCE_SURAVAGE && roadId === unsavedRoadId) {
-        return 'rgba(211, 175, 246,' + 0.65 * opacityMultiplier + ')';
+        return 'rgba(211, 175, 246,' + opacityMultiplier + ')';
```

Comment:
Not a remark but a question:
Is opacityMultiplier already adjusted to the 0.65 * opacityMultiplier due to the removal of it?

Reply:
I adjusted the opacities to look right with the opacityMultiplier. I didn't find use for those 0.65 multipliers, they just made things more difficult.

Reply:
All Right, thanks @tomixii 👍